--- origsrc/ocaml-theora-0.1.0/src/META.in	2009-02-17 12:40:01.000000000 -0600
+++ src/ocaml-theora-0.1.0/src/META.in	2009-03-04 19:50:25.742250000 -0600
@@ -1,6 +1,6 @@
 name="theora"
 version="@VERSION@"
 description="OCaml bindings for libtheora"
-requires="bigarray"
+requires="ogg bigarray"
 archive(byte)="theora.cma"
 archive(native)="theora.cmxa"
--- origsrc/ocaml-theora-0.1.0/src/Makefile.in	2009-02-17 12:40:01.000000000 -0600
+++ src/ocaml-theora-0.1.0/src/Makefile.in	2009-03-04 19:54:46.586000000 -0600
@@ -28,9 +28,10 @@ RESULT = theora
 OCAMLDOCFLAGS = -stars
 LIBINSTALL_FILES = $(wildcard *.mli *.cmi *.cma *.cmxa *.cmx *.a *.so)
 INCDIRS = @OCAMLOGG_INC@
-LIBS = bigarray
-ACLIBS = @LIBS@ @libtheora_LIBS@
-CLIBS = $(ACLIBS:-l%=%)
+LIBDIRS = @OCAMLOGG_INC@
+LIBS = ogg bigarray
+ACLIBS = -logg_stubs @LIBS@ @libtheora_LIBS@
+CLIBS = $(LIBS:-l%=%) $(ACLIBS:-l%=%)
 LDFLAGS = @LDFLAGS@ @libtheora_LDFLAGS@
 CC = @CC@
 CFLAGS = @CFLAGS@ @libtheora_CFLAGS@ -Wall -DCAML_NAME_SPACE
